kvmagent/kvmagent/test/prometheus_testsuite/test_collect_memory_overcommit_statistics.py

```
@@ -0,0 +1,20 @@
+from kvmagent.test.utils.stub import *
```

Ruff (0.11.9)

`from kvmagent.test.utils.stub import *` used; unable to detect undefined names

(F403)